doc/tutorials/run_all_tests_with_docker.md

```diff
@@ -0,0 +1,109 @@
+# Introduction
+
+Running all the tests like the build server does it requires to have multiple dependencies preinstalled. To overcome this problem, instead of trying to install all the necessary dependencies on your own, an appropriate Docker image should be used. This way you can easily and quickly run all the tests.
+
+## Who is this guide for?
+
+For anyone who wants to run all the tests, like it is done by the build server, once changes are pushed to the repository.
+
+This is a step-by-step guide. Just follow the steps and you are good to go!
+
+
+## Prerequisites
+
+- Docker for Linux containers has to be preinstalled. Please refer to https://docs.docker.com/install/ if you haven't installed it yet. Your host OS can be either Linux or Windows of course.
+- Basic knowledge of Docker (not mandatory)
+
+
+## What to begin with?
+
+### 1. Pick a Docker image and pull it
+
+Pick one of the available Docker images of Elektra. If you do not know the difference, just pick this one --> "build-elektra-debian-stretch". Unfortunately, it will take some time to download it, since it is pretty big, but you can be sure you'll have all the needed dependencies.
```

Comment:
Maybe also suggest some alpine image as alternative for slow connections.

Reply:
But I thought it was not possible to run all tests with the alpine image. If it is possible, then I can suppest using it in the tutorial.

Reply:
Yes you are right but you can also mention this limitation in the tutorial.